lib/jekyll/site.rb

```diff
@@ -279,9 +279,10 @@ def site_payload
     # Get the implementation class for the given Converter.
     # Returns the Converter instance implementing the given Converter.
     # klass - The Class of the Converter to fetch.
-
     def find_converter_instance(klass)
-      converters.find { |klass_| klass_.instance_of?(klass) } || \
+      @find_converter_instance ||= {}
+      @find_converter_instance[klass] = \
```

Comment:
I'm probably missing something but shouldn't this be `@find_converter_instance[klass] ||=` ?

Reply:
You're correct @Crunch09. Nice catch :+1: